website/events/admin/event.py

```diff
@@ -252,3 +223,14 @@ def get_urls(self):
             ),
         ]
         return custom_urls + urls
+
+    def get_field_queryset(self, db, db_field, request):
```

Comment:
This looks cool, maybe we can create an issue to do this in the sales app too? I don't think we have something like this there.

_Or_ we decide it is not really a requirement and don't do it because we made that choice there :)

Reply:
Same is true for the check in the form btw. I like it tho. Good thinking.